core/src/main/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -242,117 +301,149 @@
       IntervalViewDescriptor.create(
-          "rpc client server_elapsed_time",
+          "/grpc.io/client/server_elapsed_time/interval",
           "Minute and Hour stats for server elapsed time in msecs",
           RPC_CLIENT_SERVER_ELAPSED_TIME,
           IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
           Arrays.asList(RPC_CLIENT_METHOD));
 
+  public static final IntervalViewDescriptor RPC_CLIENT_STARTED_COUNT_INTERVAL_VIEW =
+      IntervalViewDescriptor.create(
+          "/grpc.io/client/started_count/interval",
+          "Minute and Hour stats on the number of RPC clients started",
+          RPC_CLIENT_STARTED_COUNT,
+          IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
+          Arrays.asList(RPC_CLIENT_METHOD));
+
+  public static final IntervalViewDescriptor RPC_CLIENT_FINISHED_COUNT_INTERVAL_VIEW =
+      IntervalViewDescriptor.create(
+          "/grpc.io/client/finished_count/interval",
+          "Minute and Hour stats on the number of RPC clients finished",
+          RPC_CLIENT_FINISHED_COUNT,
+          IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
+          Arrays.asList(RPC_CLIENT_METHOD));
+
   // RPC server {@link IntervalViewDescriptor}s.
   public static final IntervalViewDescriptor RPC_SERVER_SERVER_LATENCY_INTERVAL_VIEW =
       IntervalViewDescriptor.create(
-          "rpc server server_latency",
+          "/grpc.io/server/server_latency/interval",
           "Minute and Hour stats for server latency in msecs",
           RPC_SERVER_SERVER_LATENCY,
           IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
           Arrays.asList(RPC_SERVER_METHOD));
 
   public static final IntervalViewDescriptor RPC_SERVER_REQUEST_BYTES_INTERVAL_VIEW =
       IntervalViewDescriptor.create(
-          "rpc server request_bytes",
+          "/grpc.io/server/request_bytes/interval",
           "Minute and Hour stats for request size in bytes",
           RPC_SERVER_REQUEST_BYTES,
           IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
           Arrays.asList(RPC_SERVER_METHOD));
 
   public static final IntervalViewDescriptor RPC_SERVER_RESPONSE_BYTES_INTERVAL_VIEW =
       IntervalViewDescriptor.create(
-          "rpc server response_bytes",
+          "/grpc.io/server/response_bytes/interval",
           "Minute and Hour stats for response size in bytes",
           RPC_SERVER_RESPONSE_BYTES,
           IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
           Arrays.asList(RPC_SERVER_METHOD));
 
   public static final IntervalViewDescriptor RPC_SERVER_ERROR_COUNT_INTERVAL_VIEW =
       IntervalViewDescriptor.create(
-          "rpc server error_count",
-          "Minute and Hour stats for rpc errors",
+          "/grpc.io/server/error_count/interval",
+          "Minute and Hour stats for /grpc.io/errors",
```

Comment:
Is this a typo?  The measurement descriptor name above is "/grpc.io/server/error_count".